deploy/examples/migrationcontainer/README.md

```diff
@@ -22,12 +22,18 @@ Labels that should be added as groupings to the prometheus job. Migration
 containers should truncate the prefix `DBA_OP_LABEL_` and lower case any
 remainder before setting the grouping.
 
+### DBA_OP_CONNECTION_ENGINE
+
+The engine (e.g. mysql, postgres) to use for connecting to the database with the
+specified DSN.
+
 ### DBA_OP_CONNECTION_STRING
 
-A database connection string that contains the username, password, hostname, port,
+A golang database DSN that often contains the username, password, hostname, port,
 and logical database schema, e.g.:
 
-`<engine>://<username>:<password>@<hostname>:<port>/<dbname>`
+`username:password@tcp(hostname:port)/dbname`
```

Comment:
@jakedt is this for MySQL? Should it have engine similar to postgres example below? Also you need additional line break between examples or both examples will be display on same line.

Reply:
It is for MySQL. Unfortunately the DSNs don't have a lot of consistency in go, so no it doesn't have the engine. Added the blank line as well as embedding headers.